src/context/Context.js

```diff
@@ -3,22 +3,37 @@ export const Context = createContext();
 
 export const ContextProvider =props=>{
     const [items, setItems] = useState([]);
+    const [cart, setCart] = useState([]);
     const [completedOrPending, setCompletedOrPending] =useState(false);
+    const [itemDetail, setItemDetail] = useState();
 
-    const addItems =(item)=>{
-        setItems([...items,{item}]);
-    }
+    const addToCart =(item)=>{
+        setCart(...cart,{item})
+    };
     const deleteItem= id =>{
-        setItems(items.filter(item=>item.id!==id));
-    }
+        setCart(cart.filter(item=>item.id!==id));
+    };
+    const decreaseQuantity=()=>{
 
+    };
+    const increaseQuantity=()=>{
+
+    };
+    const handleItemDetail=id=>{
+        const findId= items.find(task=>task._id===id);
+        setItemDetail([findId]);
```

Comment:
This probably doesn't need to be wrapped in an array since there will only be one item "selected".